egs/multi_conv_zh_es_ta/ST/hent_srt/export.py

```
@@ -0,0 +1,640 @@
+
+        new_states = new_encoder_states + [
+            new_cached_embed_left_pad,
+            new_processed_lens,
+        ]
+        return encoder_out, encoder_out_lens, new_states
+
+    @torch.jit.export
+    def get_init_states(
+        self,
+        batch_size: int = 1,
+        device: torch.device = torch.device("cpu"),
```

Ruff (0.12.2)

Do not perform function call `torch.device` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)